src/crypto.rs

```diff
@@ -29,10 +29,8 @@ pub enum Algorithm {
 /// The actual HS signing + encoding
 fn sign_hmac(alg: &'static digest::Algorithm, key: &[u8], signing_input: &str) -> Result<String> {
     let signing_key = hmac::SigningKey::new(alg, key);
-    Ok(base64::encode_config(
-        hmac::sign(&signing_key, signing_input.as_bytes()).as_ref(),
-        base64::URL_SAFE_NO_PAD
-    ))
+    Ok(base64::encode_config(hmac::sign(&signing_key, signing_input.as_bytes()).as_ref(),
```

Comment:
Yeah, sorry about that! Turns out my rustfmt was old. Would you prefer me to avoid rustfmt altogether, or add a config like the two linked from serde & rustmft?